swap-logic/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "swap-logic"
+version = "0.1.0"
+authors = ["Bryan RHEE <bryan.rhee@hdac.io>"]
+edition = "2018"
+
+[lib]
+crate-type = ["lib", "cdylib"]
+bench = false
+doctest = false
+test = false
+
+[features]
+std = ["contract/std", "types/std"]
+lib = []
+enable-bonding = []
```

Comment:
Seems okay to delete this feature.

Reply:
https://github.com/hdac-io/swap/pull/1/commits/20729abfb6211eece9df8ae624629722b09c6b6e